src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -131,7 +131,7 @@ public void testPlugins() throws PluginCompatibilityTesterException {
                 && localCheckoutProvided()
                 && !data.plugins.containsKey(config.getIncludePlugins().get(0))) {
             String artifactId = config.getIncludePlugins().get(0);
-            UpdateSite.Plugin extracted = extractFromLocalCheckout();
+            UpdateSite.Plugin extracted = extractFromLocalCheckout(artifactId);
```

Comment:
this was broken as the plugin name would be set to the artifactId of the root of the checkout which for multimodule projects is always wrong.

as we are only testing a single pluign we know the name which is the artifactId so use it.